packages/communication-react/review/communication-react.api.md

```diff
@@ -1942,6 +1942,7 @@ export type MessageThreadProps = {
     disableEditing?: boolean;
     strings?: Partial<MessageThreadStrings>;
     fileDownloadHandler?: FileDownloadHandler;
+    dateTimeFormat?: (messageDate: Date) => string;
```

Comment:
The callback is named `dateTimeFormat` but the input parameter is `messageDate`
A better name would be `messageDateTime` as it is inline with the callback name.

Reply:
OR changing the callback name to `dateFormat` is another option